L1Trigger/TrackFindingTracklet/interface/Settings.h

```diff
@@ -13,6 +13,7 @@
 
 #include "FWCore/MessageLogger/interface/MessageLogger.h"
 #include "FWCore/Utilities/interface/Exception.h"
+#include "L1Trigger/TrackTrigger/interface/Setup.h"
```

Comment:
I'm not sure what to do about this - this dependency makes it impossible to run the standalone version of the code outside cmssw... As I go through he code I will see if I can understand how we can break this dependency.

Reply:
Actually I don't think you need to #include Setup here - you can just forward declare the class as all you use is a pointer to the class in Settings.h.